File: rust-package/brainflow/src/log_levels.rs
#[derive(Debug, Copy, Clone, Hash, PartialEq, Eq, FromPrimitive)]
pub enum LogLevels {
    LevelTrace = 0,
    #[doc = " TRACE"]
    LevelDebug = 1,
    #[doc = " DEBUG"]
    LevelInfo = 2,
    #[doc = " INFO"]
    LevelWarn = 3,
    #[doc = " WARN"]
    LevelError = 4,
    #[doc = " ERROR"]
    LevelCritical = 5,
    #[doc = " CRITICAL"]
    LevelOff = 6,
}

File: matlab_package/brainflow/LogLevels.m
classdef LogLevels < int32
    % Store all possible log levels
    enumeration
        LEVEL_TRACE(0)
        LEVEL_DEBUG(1)
        LEVEL_INFO(2)
        LEVEL_WARN(3)
        LEVEL_ERROR(4)
        LEVEL_CRITICAL(5)
        LEVEL_OFF(6)
    end
end